orderer/common/cluster/deliver_test.go

```diff
@@ -271,8 +272,9 @@ func (ds *deliverServer) addExpectProbeAssert() {
 
 func (ds *deliverServer) addExpectPullAssert(seq uint64) {
 	ds.seekAssertions <- func(info *orderer.SeekInfo, _ string) {
-		assert.NotNil(ds.t, info.GetStart().GetSpecified())
-		assert.Equal(ds.t, seq, info.GetStart().GetSpecified().Number)
+		seekSpecified := info.GetStart().GetSpecified()
```

Comment:
Let's require **info.GetStart()** not to be nil as well.

Reply:
Sure, I actually originally had a check for both but when I looked at the code for the GetStart() function it looked somewhat unlikely that it would ever return nil if info itself wasn't already nil.